tests/sps/test_article_abstract.py

```diff
@@ -0,0 +1,464 @@
+                </trans-abstract>
+            </article-meta>
+            </front>
+            <sub-article article-type="translation" id="s1" xml:lang="pt">
+                    <front-stub>
+                        <article-id pub-id-type="doi">10.1590/1980-220X-REEUSP-2021-0569pt</article-id>
+                        <abstract>
+                            <title>RESUMO</title>
+                            <sec>
+                                <title>Objetivo:</title>
+                                <p>objetivo</p>
+                            </sec>
+                            <sec>
+                                <title>Método:</title>
+                                <p>metodo</p>
+                            </sec>
+                            <sec>
+                                <title>Resultados:</title>
+                                <p>resultados</p>
+                            </sec>
+                            <sec>
+                                <title>Conclusão:</title>
+                                <p>conclusão</p>
+                            </sec>
+                        </abstract>
+                    </front-stub>
+                </sub-article>
+            </article>
+            """
+        )
+        data = ET.fromstring(xml)
+        obtained = Abstract(data).abstracts_with_tags
+
+        expected = [
+            {
+                "lang": "en",
+                "title": "Abstract",
+                "sections": {
+                    "Objective:": "objective",
+                    "Method:": "method",
+                    "Results:": "results",
+                    "Conclusion:": "conclusion"
+                }
+            },
+            {
+                "lang": "es",
+                "title": "RESUMEN",
+                "sections": {
+                    "Objetivo:": "objetivo",
+                    "Método:": "metodo",
+                    "Resultados:": "resultados",
+                    "Conclusión:": "conclusion"
+                }
+            },
+            {
+                "lang": "pt",
+                "title": "RESUMO",
+                "sections": {
+                    "Objetivo:": "objetivo",
+                    "Método:": "metodo",
+                    "Resultados:": "resultados",
+                    "Conclusão:": "conclusão"
+                }
+            }
+
+        ]
+
+        self.assertEqual(obtained, expected)
+
+    def test_abstracts_without_tags(self):
+        xml = (
+            """
+            <article xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" 
+            article-type="research-article" dtd-version="1.1" specific-use="sps-1.9" xml:lang="en">
+            <front>
+            <article-meta>
+                <abstract>
+                    <title>Abstract</title>
+                        <sec>
+                        <title>Objective:</title>
+                            <p>objective</p>
```

Comment:
Faça testes em que entre `<p>` e `</p>`, exista mais tags. Pegue resumos de artigos reais, principalmente da área de biológicas em que há os itálicos em nomes de espécies.